### DISCUSSION TITLE
Lesson 0: Rinkeby error

### DISCUSSION THREAD

**Discussion by Zeyad20080** [smartcontractkit/full-blockchain-solidity-course-py#782] [Q&A]
Why I can't get test eth by https://faucet.rinkeby.io/ 
It says that faucet is offline and I don't know why
Can anyone help

**Comment by cromewar** [comment]
Hello @Zeyad20080 is normal, that faucet goes online time to time, you can use this one: https://faucets.chain.link/
Or paste your wallet address here I'll sent you some ETH.
